master/internal/experiment.go

```diff
@@ -1003,7 +1005,7 @@ func checkpointFromTrialIDOrUUID(
 		if err != nil {
 			return nil, errors.Wrap(err, "invalid source checkpoint UUID")
 		}
-		checkpoint, err = db.CheckpointByUUID(checkpointUUID)
+		checkpoint, err = checkpoints.CheckpointByUUID(context.TODO(), checkpointUUID)
```

Comment:
question: Is there a different context you could be using here instead? Does the caller have a context that could be passed in as parameter to this function?

Reply:
The caller doesn't have a context passed into it, there is an instance of calling context.TODO() in the caller when passing into another function. I can do that here as well. Let me know what you prefer!